### ISSUE TITLE
makeFixedHoldoutInstance - used a fixed dataset from a distinct file/object? [type-question]

### ISSUE THREAD

**Issue by mlearner-jl** [mlr-org/mlr#2131]
When preparing data, I will often create a Train/Validation/Test split and store these in separate variables and files.

With the function makeFixedHoldoutInstance is there a way to specify this separate Validation set to be used in the resampling method (essentially here I am talking about the outer resampling method for a benchmark experiment).

For example perhaps there is a way at a late stage to merge the Validation set into the train task and then specify the indices?

**Comment by larskotthoff** [comment]
If I understand what you're trying to do correctly the answer is no, but it sounds like you would be able to do this by explicitly creating a nested resampling.

**Comment by mlearner-jl** [comment]
Yes I am trying to use nested resampling
For the inner method I am using CV on the train data; I was hoping to use a Holdout method for the outer resampling for the benchmark experiment with a separate validation set.

Does that make sense?

**Comment by mlearner-jl** [comment]
What I am trying to avoid, is when for example downsampling is used to account for say class imbalance, I want to leave the Validation set untouched. Equally I do not want the Validation set to be part of the cross-validation.

I imagine blocking could be used here perhaps?

**Comment by mlearner-jl** [comment]
I have been able to do what I need.
After performing various manipulations (downsampling etc.) using the training data (as part of a train task), I extract the data using getTaskData(), rbind the validation set and create a new task with the validation data also. I then create a fixedHoldoutInstance and can do holdout on the specified validation set for my outer resampling strategy.

Thanks for your help
